assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -39,11 +40,15 @@ const CartEditor = ( { className, attributes, setAttributes } ) => {
 						'woo-gutenberg-products-block'
 					) }
 					checked={ isShippingCalculatorEnabled }
-					onChange={ () =>
+					onChange={ () => {
+						const newEnabled = ! isShippingCalculatorEnabled;
```

Comment:
I don't think this const declaration is necessary. You can just use `! isShippingCalculatorEnabled` in both places it is used.

Reply:
I've made that change here: b7e11874b95b14ca018b3ae2bf14d50b86cbffbf